app/components/object-inspector/property.ts

```diff
@@ -122,7 +122,10 @@ export default class ObjectInspectorProperty extends Component<ObjectInspectorPr
 
   get cannotEdit() {
     if (this.args.model.name === '...' || !this.isCalculated || this.readOnly) return true;
-    return this.args.model?.value?.type !== 'type-string' && this.args.model?.value?.type !== 'type-number';
+    return
+      this.args.model?.value?.type !== 'type-string' &&
+        this.args.model?.value?.type !== 'type-number' &&
+        this.args.model?.value.type !== 'type-boolean';
```

Comment:
```suggestion
    return !['type-string', 'type-number', 'type-boolean'].includes(this.args.model?.value?.type);
```

Perhaps we could simplify this a bit?